docs/manual/install/arch.rst

```diff
@@ -20,39 +20,33 @@ if you use `yaourt`_:
     # or for develop
     yaourt -S qtile-python3-git
 
-Using pacman
-============
+Using makepkg
+=============
 
-.. code-block:: bash
-
-    sudo pacman -S python pango python-cairocffi python-xcffib
-
-Also you need one qtile package from the AUR:
-
-- `qtile-python3-git`_ 
-- `qtile`_ 
+The latest version of either package can be obtained by downloading a snapshot
+or cloning its repository:
 
+.. code-block:: bash
 
-Installing AUR packages without helper
-======================================
+    # snapshot
+    curl -s https://aur.archlinux.org/cgit/aur.git/snapshot/<package-name>.tar.gz | tar -xvzf -
+    # or repository
+    git clone https://aur.archlinux.org/<package-name>.git
```

Comment:
This looks a bit inconsistent with the section above, which uses the actual package names instead of the `<package-name>` placeholder.